### PR TITLE
Microsoft.Sysinternals.ProcessMonitor version 3.92

### PR DESCRIPTION
**Pull request by mivsvit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92821)

### CODE REVIEW

File: manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.92/Microsoft.Sysinternals.ProcessMonitor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.Sysinternals.ProcessMonitor
+PackageVersion: 3.92
+MinimumOSVersion: 10.0.0.0
+Commands:
+- process-monitor
+Installers:
+- Architecture: x86
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.92/Microsoft.Sysinternals.ProcessMonitor.installer.yaml:11]:
```suggestion
  InstallerType: portable
```

File: manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.92/Microsoft.Sysinternals.ProcessMonitor.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.Sysinternals.ProcessMonitor
+PackageVersion: 3.92
+MinimumOSVersion: 10.0.0.0
+Commands:
+- process-monitor
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.92/Microsoft.Sysinternals.ProcessMonitor.installer.yaml:10]:
```suggestion
- Architecture: x64
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Please also remove version 3.91 in this PR

**Review by stephengillie — APPROVED** [review]

**Pull request merged**